custom/icds/tasks/data_pulls.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+from django.conf import settings
+from django.core.management import call_command
+from django.urls import reverse
+from django.utils.translation import ugettext_lazy as _
+
+from celery.task import task
+
+from couchexport.models import Format
+from dimagi.utils import web
+from soil.util import expose_blob_download
+
+from corehq.apps.hqwebapp.tasks import send_html_email_async
+from corehq.util.files import safe_filename_header
+
+
+@task(queue='icds_dashboard_reports_queue')
+def run_data_pull(data_pull_slug, month, location_id=None, email=None):
+    subject = _('Custom ICDS Data Pull')
+    try:
+        filename = call_command("run_custom_data_pull", data_pull_slug, "icds-ucr-citus", month=month,
+                                location_id=location_id, skip_confirmation=True)
+    except Exception as e:
+        if email:
+            message = _("""
+                            Hi,
+                            Could not generate the requested data pull.
+                            The error has been notified. Please report as an issue for quicker followup
+                        """)
+            send_html_email_async.delay(subject, [email], message,
+                                        email_from=settings.DEFAULT_FROM_EMAIL)
+        raise e
```

Comment:
nit: I think `e` is redundant here

Reply:
you mean just using raise would do because now python3?

Reply:
`raise` works in python2 as well, but yeah i think is clearer.